### PR TITLE
Update de.fau.rrze.NetworkShareMounter.plist

### PR DESCRIPTION
**Pull request by Ci7rix**

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist [hunk 1]
```diff
@@ -149,16 +209,72 @@ A profile can consist of payloads with different version numbers. For example, c
...
+			<string>kerberosRealm</string>
+			<key>pfm_title</key>
+			<string>Kerberos Realm</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_app_min</key>
+			<string>2</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist:237]: Are you sure that these new subkeys `keychainiCloudSync`, `keyChainService`, and `keyChainComment` were already available in version 2.0.0?

File: Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist [hunk 2]
```diff
@@ -247,9 +365,11 @@ A profile can consist of payloads with different version numbers. For example, c
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>Array with all network shares. Example: smb://filer.your.domain/share. Note: %USERNAME% will be replaced with the login name of the current user.</string>
+			<string>⚠️ Deprecated with version 3. Value will be removed in a future version of the Network Share Mounter.</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist:368]: For reference and for users on older versions, we usually keep the description of deprecated keys. Could you please revert the changes to this line?

You can then mark the key as deprecated by using `pfm_app_deprecated` set to `3.0.0`.

**Review by relgit — CHANGES_REQUESTED** [review]
Thanks for the PR @Ci7rix.

Please see my comments.

**Comment by kevinmcox** [comment]
@Ci7rix just a bump to see if you'll be able to address the notes above. Thanks!

**Comment by Ci7rix** [comment]
Sorry, I was AFK.I will have a look tomorrow or after tomorrow. Le 2 sept. 2024 à 19:18, Kevin M. Cox ***@***.***> a écrit :﻿
@Ci7rix just a bump to see if you'll be able to address the notes above. Thanks!

File: Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist (hunk 1 above)
Comment (Ci7rix) [reply Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist]: I couldn't find information about those keys in the README so I removed them.

File: Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist (hunk 2 above)
Comment (relgit) [reply Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist]: Please also make sure to add a `pfm_app_deprecated` tag set to `3.0.0`, thanks!

Comment (relgit) [reply Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist]: My bad, I see now that it was already added previously.
